src/main/scala/package.scala

```diff
@@ -1,21 +1,24 @@
 package com.originate.scalypher
 
+import com.originate.scalypher.action.ActionNodeOrRelationship
 import com.originate.scalypher.action.ActionReference
 import com.originate.scalypher.where.ObjectReference
-import com.originate.scalypher.where.Reference
+import com.originate.scalypher.where.NodeOrRelationshipReference
 
 package object types {
-  trait Referenceable {
-    def as(name: String): ActionReference =
-      ActionReference(ObjectReference(this), Some(name))
+  trait Referenceable
```

Comment:
yeah but you can't extend a sealed trait unless that trait is defined in the same file. so i would have to put the node/relationship definition in the same file. i guess i could do that? seems wonky though.